crates/types/src/interpolated.rs

```diff
@@ -18,10 +21,10 @@ impl Interpolated {
     const ARGUMENT_FIRST_HALF_OPPONENT_HALF_TOWARDS_OWN_GOAL: Point2<f32> = point![3.0, 0.0];
     const ARGUMENT_FIRST_HALF_OPPONENT_HALF_AWAY_OWN_GOAL: Point2<f32> = point![3.0, PI];
 
-    pub fn evaluate_at(&self, robot_to_field: Isometry2<f32>) -> f32 {
+    pub fn evaluate_at(&self, ground_to_field: Transform<Ground, Field, Isometry2<f32>>) -> f32 {
```

Comment:
Use new typedef

Reply:
Done, also cleaned up the tests in this file w.r.t. using nalgebra types.